spec/features/campaign_overview_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe 'campaign overview page', type: :feature, js: true do
+  let(:slug)  { 'spring_2016' }
+  let(:user) { create(:user) }
+  let(:admin) { create(:admin) }
+  let(:campaign) {
+    create(:campaign,
+           id: 10001,
+           title: 'My awesome Spring 2016 campaign',
+           slug: slug,
+           description: 'This is the best campaign')
+  }
+
+  before do
+    Capybara.current_driver = :poltergeist
+    stub_token_request
```

Comment:
`stub_token_request` is for when the system tries to get a csrf token from Wikipedia. We don't need that here.